backend/src/main/java/saci/infrastructure/LevelRepository.java

```diff
@@ -10,10 +12,16 @@
 public interface LevelRepository extends JpaRepository<Level, Long> {
     Optional<Level> findByName(String name);
 
+    List<Level> findByRoleId(Long roleId);
+
     @Query(
             "SELECT COUNT(*) FROM Level l WHERE l.roleId = :roleId AND (l.minCoefficient < :maxCoefficient AND l.maxCoefficient > :minCoefficient)")
     int overlappingLevelsCounter(
             @Param("roleId") Long roleId,
             @Param("minCoefficient") Integer minCoefficient,
             @Param("maxCoefficient") Integer maxCoefficient);
+
+    @Query(
+            "SELECT l FROM Level l WHERE l.roleId = :roleId ORDER BY l.minCoefficient ASC, l.maxCoefficient ASC")
+    List<Level> findByRoleIdOrderByMinCoefficientAscMaxCoefficientAsc(@Param("roleId") Long roleId);
```

Comment:
Change this method name to a simpler one please, it's hard to understand easily why you do this.